src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ExperimentalFeature/index.html.twig

```diff
@@ -0,0 +1,72 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain 'Admin.Advparameters.Feature' %}
+
+{% block content %}
+  <div class="row justify-content-center">
+    {{ form_start(featureFlagsForm, {attr : {class: 'form col', id:'feature-flag-form'}, action: path('admin_experimental_feature_save') }) }}
+    {% block experimental_feature_form %}
+      <div class="card" id="configuration_fieldset_general">
+        <h3 class="card-header">
+          <i class="material-icons">settings</i> {{ 'Experimental features'|trans }}
+        </h3>
+
+        <div class="alert medium-alert alert-warning" role="alert">
+          {{ 'Testing a feature before its official release can be exciting. However, you must be aware of the potential risks of such experiments:'|trans({}, 'Admin.Advparameters.Notification') }}
+          <ul>
+            <li>{{ 'Experimental features are still under development. Enabling them could therefore have unintended consequences and cause data loss.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+            <li>{{ 'In any case, you should never experiment in production.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+          </ul>
+        </div>
```

Comment:
You need to move this block inside "card-text" below.

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/326c2fd2e529fa0f63d2a53b60beb877bdb647e7